test/unit/insights.test.js

```diff
@@ -0,0 +1,41 @@
+describe('application insights', () => {
+  const DEFAULT_ENV = process.env
+  let applicationInsights
+
+  beforeEach(() => {
+    // important to clear the cache when mocking environment variables
```

Comment:
Do we need this comment?

Reply:
I thought it would be helpful to add that comment as it's a bit of a gotcha.  You could argue as this comment now has been copied into about 20 repos it's probably a bit redundant.

Removed.